src/components/MultiHttp/consts.ts

```diff
@@ -0,0 +1,63 @@
+import isEmpty from 'lodash';
+import { UseFormGetValues, FieldValues } from 'react-hook-form';
+import { CheckType, Check, AlertSensitivity, MultiHttpSettings } from 'types';
+import { MultiHttpEntry, KeyTypes } from 'components/MultiHttp/MultiHttpTypes';
+import { fallbackSettings } from 'components/constants';
+
+export const multiHttpFallbackCheck = {
+  job: '',
+  target: '',
+  frequency: 120000,
+  timeout: 3000,
+  enabled: true,
+  labels: [],
+  probes: [],
+  alertSensitivity: AlertSensitivity.None,
+  settings: {
+    multihttp: fallbackSettings(CheckType.MULTI_HTTP) as MultiHttpSettings,
+  },
+  basicMetricsOnly: true,
+} as Check;
+
+const removeEmptyFields = (getValues: any) => {
+  const entries = getValues().settings.multihttp;
+  getValues().settings.multihttp.entries.forEach((ent: MultiHttpEntry) => {
```

Comment:
I feel like it's worth digging into how we're using the form to make sure this is necessary. If it is, then 🤷‍♂️